Add examples of individually setting bits in Physics introduction (#9910)
godotengine/godot-docs: tutorials/physics/physics_introduction.rst

In function calls, layers are specified as a bitmask. Where a function enables
all layers by default, the layer mask will be given as ``0xffffffff``. Your code

> can use binary, hexadecimal, or decimal notation for layer masks depending

Reviewer: Please restore, unrelated and an appropriate pause

    # (2^(1-1)) + (2^(3-1)) + (2^(4-1)) = 1 + 4 + 8 = 13
    pow(2, 1-1) + pow(2, 3-1) + pow(2, 4-1)

> You can also set bits independently by calling 'set_collision_layer_value(layer_number, value)'

Reviewer: ```suggestion
You can also set bits independently by calling ``set_collision_layer_value(layer_number, value)``
```

> or 'set_collision_mask_value(layer_number, value)' on any given :ref:`CollisionObject <class_CollisionObject2D>` as follows::

Reviewer: ```suggestion
or ``set_collision_mask_value(layer_number, value)`` on any given :ref:`CollisionObject <class_CollisionObject2D>` as follows::
```

> or ``set_collision_mask_value(layer_number, value)`` on any given :ref:`CollisionObject <class_CollisionObject2D>` as follows::

Reviewer 2: ```suggestion
or ``set_collision_mask_value(layer_number, value)`` on any given :ref:`CollisionObject2D <class_CollisionObject2D>` as follows::
```
Use the whole name

>     # Example: Setting mask value for enabling layers 1, 3 and 4

Reviewer: ```suggestion
    # Example: Setting mask value for enabling layers 1, 3, and 4.
```

    var collider: CollisionObject2D = $CollisionObject2D  # Any given collider.

>     collider.set_mask_layer_value(1, true)
>     collider.set_mask_layer_value(3, true)
>     collider.set_mask_layer_value(4, true)

Reviewer 2: ```suggestion
    collider.set_collision_mask_value(1, true)
    collider.set_collision_mask_value(3, true)
    collider.set_collision_mask_value(4, true)
```
The function name was wrong in the example, there is no `set_mask_layer_value()` function
